rbac/rbac/settings.py

```diff
@@ -315,3 +319,13 @@
 # Optionally log all DB queries
 if ENVIRONMENT.bool("LOG_DATABASE_QUERIES", default=False):
     LOGGING["loggers"]["django.db.backends"] = {"handlers": ["console"], "level": "DEBUG", "propagate": False}
+
+# Internal API Configuration
+INTERNAL_API_PATH_PREFIXES = ["/_private/"]
+
+try:
+    INTERNAL_DESTRUCTIVE_API_OK_UNTIL = parse_dt(
+        os.environ.get("RBAC_DESTRUCTIVE_ENABLED", "not-a-real-time"), tzinfos={"UTC": pytz.UTC}
```

Comment:
You may want to consider ensuring that this is a tz-aware datetime coming out of here, versus a naive one.

Reply:
Ah, for some reason I thought I'd tested this in a way that did ensure that, but that was only if you set the tz in the config. Pushed a commit that should resolve that, thanks!